indico/modules/designer/controllers.py

```diff
@@ -281,9 +297,16 @@ def _process_GET(self):
     def _process_POST(self):
         data = dict({'background_position': 'stretch', 'items': []}, **request.json['template'])
         self.validate_json(TEMPLATE_DATA_JSON_SCHEMA, data)
-        invalid_placeholders = {x['type'] for x in data['items']} - set(get_placeholder_options()) - {'fixed'}
+        invalid_placeholders = {x['type'] for x in data['items']} - set(get_placeholder_options())
         if invalid_placeholders:
             raise UserValueError('Invalid item types: {}'.format(', '.join(invalid_placeholders)))
+        image_items = [item for item in data['items'] if item['type'] == 'fixed_image']
+        template_images = ({img.id for img in self.template.images})
```

Comment:
Not sure why flake8 didn't complain about this, but those parentheses are certainly not needed...

```suggestion
        template_images = {img.id for img in self.template.images}
```